ts/screens/onboarding/FingerprintScreen.tsx

```diff
@@ -14,69 +14,91 @@ import {
   fingerprintAcknowledge
 } from "../../store/actions/onboarding";
 
-import { BiometrySimpleType } from "../../sagas/startup/checkSupportedFingerprintSaga";
+import { BiometrySimpleType } from "../../sagas/startup/checkAcknowledgedFingerprintSaga";
 import { Dispatch } from "../../store/actions/types";
 
-type FingerprintScreenNavigationParams = {
+type NavigationParams = {
   biometryType: BiometrySimpleType;
 };
 
-type OwnProps = NavigationScreenProps<FingerprintScreenNavigationParams>;
+type OwnProps = NavigationScreenProps<NavigationParams>;
 
 type Props = OwnProps & ReturnType<typeof mapDispatchToProps>;
 
 type State = {
-  showAbortOnboardingModal: boolean;
-  biometryType?: BiometrySimpleType;
+  isAbortOnboardingModalVisible: boolean;
 };
 
 /**
- * A screen to show the if the fingerprint is supported to the user.
+ * A screen to show if the fingerprint is supported to the user.
  */
 export class FingerprintScreen extends React.PureComponent<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      showAbortOnboardingModal: false,
-      biometryType: props.navigation.getParam("biometryType")
+      isAbortOnboardingModalVisible: false
     };
   }
 
-  // The Body of this view
-  public renderBody(biometryType: BiometrySimpleType | undefined) {
-    if (biometryType === "NotEnrolled") {
+  private renderBiometryType(
+    biometryType: BiometrySimpleType
+  ): string | undefined {
```

Comment:
the check should be exhaustive and this function should never return `undefined` ([example](https://github.com/teamdigitale/italia-app/blob/master/ts/sagas/identification.ts#L63))

Reply:
fixed